src/scores/probability/crps_impl.py

```diff
@@ -174,35 +174,39 @@ def crps_cdf(
     is used as an index for predictive CDF values. Various techniques are used to
     interpolate CDF values between indexed thresholds.
 
-    Given
-        - a predictive CDF `fcst` indexed at thresholds by variable x,
-        - an observation in CDF form `obs_cdf` (i.e., obs_cdf(x) = 0 if x < obs and 1 if x >= obs),
+    Given:
+        - a predictive CDF `fcst` indexed at thresholds by variable x
+        - an observation in CDF form `obs_cdf` (i.e., obs_cdf(x) = 0 if x < obs and 1 if x >= obs)
```

Comment:
Why did you remove the comma on the first two dot points, but not the last?

Reply:
Just an oversight here, thanks, I've remove the extra comma for now. If you'd prefer them all back, that's fine too, just let me know.